python/lsst/coadd/chisquared/coadd.py

```diff
@@ -51,16 +56,23 @@ def __init__(self, bbox, wcs, badMaskPlanes, logName="coadd.chisquared.Coadd"):
     def addExposure(self, exposure, weightFactor=1.0):
         """Add a an exposure to the coadd; it is assumed to have the same WCS as the coadd
 
-        Inputs:
-        @param[in] exposure: Exposure to add to coadd; this must be:
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            Exposure to add to coadd; this must be:
             - background-subtracted or background-matched to the other images being coadded
             - psf-matched to the desired PSF model (optional)
             - warped to match the coadd
-        @param[in] weightFactor: weight with which to add exposure to coadd
+        weightFactor : `float`
+            weight with which to add exposure to coadd
```

Comment:
Capitalize Weight, and put a full-stop at the end of the sentence.

`exposure` in double back ticks.